templates/patroni.yml.j2

```diff
@@ -11,6 +11,9 @@ scope: {{ scope }}
 namespace: /db/
 name: {{ member_name }}
 
+log:
+  dir: /var/log/postgresql
```

Comment:
Was this not default? Is this log rotate? How big can it get?
(I've saw log files filling up disk and disrupting services too many times...)

Reply:
No. It was not default. When the directory is added to the configuration, Patroni will retain 4 25MB logs by default. There is a note about that in https://patroni.readthedocs.io/en/latest/SETTINGS.html.